lib/buildsystems/ruby.rb

```diff
@@ -15,7 +15,7 @@ def self.install
     @gem_ver = version.split('-', 2).first
     @ruby_ver = version.split('-', 3).last
     system "yes | gem uninstall -a #{@gem_name}", exception: false
-    system "gem install -i #{CREW_DEST_LIB_PREFIX}/ruby/gems/#{@ruby_ver}.0 -n #{CREW_DEST_PREFIX}/bin -N -v #{@gem_ver} #{@gem_name}", exception: false
+    system "gem install -i #{CREW_DEST_LIB_PREFIX}/ruby/gems/#{@ruby_ver}.0 -n #{CREW_DEST_PREFIX}/bin -N #{@gem_name}", exception: false
```

Comment:
Why remove `@gem_ver`?

Reply:
Without that, the most current version gets installed. This is how our pip installs work.

Reply:
So this means, potentially, the package version might not match the actual installed version?

Reply:
What happens in the case where you might want to lock a version if the most recent version causes an issue?

Reply:
We can modify `buildtools/ruby.rb` if that's ever an issue... but it certainly hasn't been an issue for any of the pip packages, and this has in fact hurt us by letting us miss a rubocop update that would have let us avoid an issue with the previous version of rubocop where it failed to check the glibc files properly.

Reply:
(Having said that, we desperately need a script that automatically updates versions in package files and automatically creates PRs based on version updates.)

Reply:
Next task on the list?